Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,13 +527,54 @@ bool isFunctionJson(string fullName)
             }
         }
 
-        private async Task<string> DeployZipLocally(ZipDeploymentInfo zipDeploymentInfo, ITracer tracer)
+        //Used by OneDeploy to fetch artifacts that are not ZIP files. OneDeploy ZIP files are handled with 
+        //LocalZipHandler()
```

Comment:
Add space after //
Check elsewhere too.

// OneDeploy Fetch handler for non-zip artifacts.
// For zip files, OneDeploy uses the LocalZipHandler Fetch handler